xml/System.Reflection.PortableExecutable/DllCharacteristics.xml

```diff
@@ -248,7 +248,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Reserved.</summary>
       </Docs>
     </Member>
     <Member MemberName="TerminalServerAware">
```

Comment:
Do we have a description for TerminalServerAware?

Reply:
Not in the source comments: [src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEFileFlags.cs#L113](https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEFileFlags.cs#L113)

This enum value was introduced in the very first commit of .NET Core: https://github.com/dotnet/corefx/commit/f11305d31a07ef9a2513332edc665b257cc9c5e5

So the API is as old as Framework. But judging by the value, and a quick search I did for these terms, this is my proposed description, @rpetrusha:

```The image is Terminal Server aware.```

@GrabYourPitchforks @steveharter as the area owners, is this description good enough for you?

Reply:
Seems reasonable to me, but I am not an expert.